src/showroom_connector/signals.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db.models.signals import post_save
+
+from user_preferences.models import UserPreferencesData, UserSettingsValue
+
+from .rq import enqueue
+from .sync import push_user
+
+
+def handle_post_save(sender, instance, *args, **kwargs):
```

Comment:
we could rewrite the code a bit:

```
from django.dispatch import receiver

@receiver(post_save, sender=UserPreferencesData, dispatch_uid='post_save_userpreferencesdata')
@receiver(post_save, sender=UserSettingsValue, dispatch_uid='post_save_usersettingsvalue')
def handle_post_save(sender, instance, *args, **kwargs):
    …
```

because (at least for me) the decorator style is a bit more readable